src/constants/tokenlists.ts

```diff
@@ -1,49 +1,78 @@
-// Compliant list for exchange
-export const TOKEN_LIST_DEFAULT =
-  'https://storageapi.fleek.co/balancer-team-bucket/assets/listed.tokenlist.json';
-
-// Extended list to include LBP tokens
-export const VETTED_TOKEN_LIST =
-  'https://storageapi.fleek.co/balancer-team-bucket/assets/vetted.tokenlist.json';
-
-export const TOKEN_LISTS: string[] = [
-  TOKEN_LIST_DEFAULT,
-  'ipns://tokens.uniswap.org',
-  'tokenlist.zerion.eth',
-  'tokens.1inch.eth',
-  'tokenlist.aave.eth',
-  'https://tokens.coingecko.com/uniswap/all.json',
-  'https://umaproject.org/uma.tokenlist.json'
-];
+import useConfig from '@/composables/useConfig';
 
+const { env, networkConfig } = useConfig();
 export const ETHER = {
   id: 'ether',
-  name: 'Ether',
+  name: networkConfig.nativeAssetLong,
   address: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
-  symbol: 'ETH',
+  symbol: networkConfig.nativeAsset,
   decimals: 18,
   logoURI:
     'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2/logo.png'
 };
```

Comment:
Instead of mixing in the network config here, maybe we should move the native asset definition into the configs, wdyt?

Reply:
Yeah, I think that's sensible. Gimme a couple mins.